### PR TITLE
Create CCC_TAC_Minutes-2022-08-11.md

### PR DESCRIPTION
**Pull request by krtaylor**

### CODE REVIEW

**Comment by dcmiddle** [comment]
The new Google representatives are listed, but I think it would be worth noting explicitly in the minutes that Google has updated its representatives.

File: TAC/Meetings/2022/2022-08-11/CCC_TAC_Minutes-2022-08-11.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+* **CCC YouTube**: [youtube.confidentialcomputing.io](https://youtube.confidentialcomputing.io)
+* **LFX**: [lfx.linuxfoundation.org](https://lfx.linuxfoundation.org)
+* **Join the CCC**: [join.confidentialcomputing.io](https://join.confidentialcomputing.io)
+* **Contact the CCC**: [confidentialcomputing.io/contact-us/](https://confidentialcomputing.io/contact-us/)
+
+## Agenda and Minutes
+
+### Welcome
+* Dave Thaler (DT) opened the call at 7:05am PT
+* DT Welcomed the members of the TAC and reviewed the values of the CCC and the antitrust policy of the Linux Foundation
```
Comment (dthaler) [inline TAC/Meetings/2022/2022-08-11/CCC_TAC_Minutes-2022-08-11.md:31]:
```suggestion
* DT welcomed the members of the TAC and reviewed the values of the CCC and the antitrust policy of the Linux Foundation 
```

File: TAC/Meetings/2022/2022-08-11/CCC_TAC_Minutes-2022-08-11.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+* Helene Khaykovich (JP Morgan Chase)
+* Ionut Mihalcea (Arm)
+* Jethro Beekman (Fortanix)
+* Mark F. Novak (JP Morgan Chase)
+* Matthieu Legre (Cysec)
+* Mike Bursell (Profian)
+* Mona Vij (Intel)
+* Penglin Yang (China Mobile)
+* Shalom Shefa (Cisco)
+* Simon Frost (Microsoft)
```
Comment (dthaler) [inline TAC/Meetings/2022/2022-08-11/CCC_TAC_Minutes-2022-08-11.md:67]:
```suggestion
* Simon Frost (Arm)
```

File: TAC/Meetings/2022/2022-08-11/CCC_TAC_Minutes-2022-08-11.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+* DT reviewed open issues, asked for volunteers
+
+	**ACTION:** HL add Penglin (GitHub ID: PenglinYang) and Alec Fernandez (GitHub ID: AlecFernandez) And Ananya Garg (GitHub ID: angarg05) as assignee with Dan Middleton on Issue #125 (Done)
+
+### TEEP Use case for Confidential Computing in Network
+* Penglin Yang discussed status, asked to please comment
+* DT reviewed history
+
+###Testing Hardware 
+* Enarx seeking to increase the project test budget to 10K for hardware, vote over email - quorum was not reached, didn’t vote
```
Comment (dthaler) [inline TAC/Meetings/2022/2022-08-11/CCC_TAC_Minutes-2022-08-11.md:143]:
```suggestion
* Enarx seeking to increase the project test budget to $10K for hardware, vote over email - quorum was not reached, didn’t vote
```

**Review by dthaler — APPROVED** [review]

File: TAC/Meetings/2022/2022-08-11/CCC_TAC_Minutes-2022-08-11.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+## Agenda and Minutes
+
+### Welcome
+* Dave Thaler (DT) opened the call at 7:05am PT
+* DT Welcomed the members of the TAC and reviewed the values of the CCC and the antitrust policy of the Linux Foundation 
+* Helen Lau (HL) captured the meeting minutes
+* DT reviewed the agenda and asked for any additions
+
+### Roll Call / Attendance
+#### Voting members of the TAC
```
Comment (krtaylor) [inline TAC/Meetings/2022/2022-08-11/CCC_TAC_Minutes-2022-08-11.md:36]: The TAC welcomes 2 new representatives from Google - Cfir Cohen and Catalin Sandu

File: TAC/Meetings/2022/2022-08-11/CCC_TAC_Minutes-2022-08-11.md [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+* Mark F. Novak (JP Morgan Chase)
+* Matthieu Legre (Cysec)
+* Mike Bursell (Profian)
+* Mona Vij (Intel)
+* Penglin Yang (China Mobile)
+* Shalom Shefa (Cisco)
+* Simon Frost (Arm)
+* Xinxin Fan (Iotex)
+* Helen Lau (Linux Foundation)
+
```
Comment (krtaylor) [inline TAC/Meetings/2022/2022-08-11/CCC_TAC_Minutes-2022-08-11.md:70]: The TAC would like to announce 2 new representatives from Google, welcome Cfir Cohen and Catalin Sandu!

Comment (krtaylor) [inline TAC/Meetings/2022/2022-08-11/CCC_TAC_Minutes-2022-08-11.md:70]:
```suggestion

The TAC would like to announce 2 new representatives from Google, welcome Cfir Cohen and Catalin Sandu!
```

**Pull request merged**